en/docs/integration-guides/event-integration/kafka/kafka-request-reply-pattern-with-azure-event-hub.md

```
@@ -0,0 +1,724 @@
+- A Reply Service with event-driven message processing
+- Secure communication with Azure Event Hub using SASL_SSL
+
+## Prerequisites
+
+Before starting this tutorial, ensure you have:
+
+- VS Code with WSO2 Integrator: BI and Ballerina plugins installed
+- Azure account with an active subscription ([create free account](https://azure.microsoft.com/free/))
+- Azure CLI installed (installation instructions below)
+- Basic understanding of:
+    - Publish-subscribe messaging patterns
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)